pkg/lint/lintersdb/manager.go

```diff
@@ -417,6 +417,13 @@ func (m Manager) GetAllSupportedLinterConfigs() []*linter.Config {
 		linter.NewConfig(golinters.NewForbidigo(forbidigoCfg)).
 			WithSince("v1.34.0").
 			WithPresets(linter.PresetStyle).
+			// Strictly speaking, the additional
+			// information is only needed when
+			// forbidigoCfg.AnalyzeTypes is chosen by the
+			// user. But we don't know that here in all
+			// cases (sometimes config is not loaded), so
+			// we have to assume that it is needed to be on
+			// the safe side.
```

Comment:
minor: I'm curious why this comment spreads over 7 lines of code. I think it can get in 3 lines (because of `lll = 140`)
```suggestion
			// Strictly speaking, the additional information is only needed when forbidigoCfg.AnalyzeTypes
			// is chosen by the user. But we don't know that here in all cases (sometimes config is not loaded),
			// so we have to assume that it is needed to be on the safe side.
```

Reply:
I am wrapping at a column that typically is preferred by the various projects that I work in. If golangci-lint accepts longer lines or it simply fits better into the code because it already has longer lines, then I can certainly wrap differently.

Reply:
I prefer https://sembr.org/

Reply:
4 lines

```go
// Strictly speaking,
// the additional information is only needed when forbidigoCfg.AnalyzeTypes is chosen by the user.
// But we don't know that here in all cases (sometimes config is not loaded),
// so we have to assume that it is needed to be on the safe side.
```

Reply:
Thanks. I've updated this comment and also some others to match this style.